todo_list_api/api.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from flask import Flask, request
+
+from todo_list_api.controller import TodoListController
+from todo_list_api.exceptions import HTTPException
+
+app = Flask(__name__)
+
+
+@app.errorhandler(HTTPException)
+def handle_error(error):
+    return {
+        "statusCode": error.status_code,
+        "body": error.reason
+    }
+
+
+controller = TodoListController()
+
+
+@app.route("/")
+def landing_page():
+    return {
+        "statusCode": 200,
+        "body": "Hello, welcome to the Todo list app!"
+    }
+
+
+@app.route("/lists", methods=["GET"])
+def get_lists():
+    lists = controller.get_lists()
+    return {
+        "statusCode": 200,
+        "body": [list_.dict() for list_ in lists]
+    }
+
+
+@app.route("/lists", methods=["POST"])
+def create_list():
+    data = json.loads(request.data.decode("utf-8"))
+    controller.create_list(data["name"], [])
+    return {
+        "statusCode": 200,
+        "body": "Created"
+    }
+
+
+@app.route("/lists/<list_name>", methods=["GET"])
+def get_list(list_name: str):
+    return {
+        "statusCode": 200,
```

Comment:
je geeft hardcoded 200 door, ook als je functie bv 404 terug geeft?

Reply:
404 wordt gehandled door de exception handler in de API zelf